CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ This component adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.h
 
 ## [Unreleased](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/compare/v0.6.0...HEAD)
 
+- Updated `ResourceConfigurator.cs` to set a fallback service name if one is not
```

Comment:
The `CHANGELOG.md` is intended for users, the details of the implementation, e.g.: source file name, are, in general, not relevant for users. This should move to the `## Added` section, here is my wording suggestion (@theletterf is our expert here):

```suggestion
- Added a fallback service name if one is not
```